src/picongpu/include/traits/PICToAdios.tpp

```diff
@@ -32,7 +34,7 @@ namespace traits
 {
     /** Trait for int */
     template<>
-    struct PICToAdios<int>
```

Comment:
Using them would simply not compile if someone was to forget this. So I see no need for that description